verifiers/internal/gha/slsaprovenance/v1.0/base.go

```diff
@@ -186,3 +193,12 @@ func (p *provenanceV1) GetSystemParameters() (map[string]any, error) {
 
 	return sysParams, nil
 }
+
+// GetExternalParameters() implements Provenance.GetExternalParameters.
+func (p *provenanceV1) GetExternalParameters() (map[string]interface{}, error) {
```

Comment:
Should this bit just be very well defined? We know the format of the provenance (is it fixed?), should we just build out a struct from the json?

Reply:
I think so too, but part of slsa-verifier tries to do is make sure that the provenance doesn't contain things that aren't also in the certificate. For now, I want to follow the rest of the repo's style, but in the future, maybe we could do something with j`son.RawMessage` to define the expected format, but still preserve unknown extra fields.

Reply:
sgtm. Can we create an issue in the repo as a todo (is there one already?)?

Reply:
Thanks for bringing it up. I just made the issue
 - https://github.com/slsa-framework/slsa-verifier/issues/778